Графика для всех страниц

d7add9 opened the pull request:
в разметку добавлены контентные изображения, все прочие также сохранены в папке images до лучших дней

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/1954051/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/1954051-sedona-34/3/

d7add9 committed:
добавляет контетные изображения в разметку

в разметку добавлены контентные изображения, все прочие также сохранены в папке images до лучших дней

Pakistanka commented on index.html:
```diff
@@ -10,7 +10,7 @@
     <nav class="navigation">
       <a class="navigation-logo" href="index.html">
-        <img class="page-header-logo" src="#" alt="Логотип Седоны.">
+        <img class="page-header-logo" src="images/logo.svg" width="139" height="70"" alt=" Логотип Седоны.">
```
Пробел перед "Логотип Седоны" не нужен

Pakistanka commented on index.html:
```diff
@@ -182,7 +183,8 @@ <h2 class="visually-hidden">Контакты</h2>
     <section class="footer-contacts-logo">
       <h2 class="visually-hidden">Ссылка на сайт Академии</h2>
-      <a href="https://htmlacademy.ru/"> <img src="#" alt="Логотип HTML Academy."></a>
+      <a href="https://htmlacademy.ru/"> <img src="images/logo-htmlacademy.svg" width="115" height="34"
```
Мне не нравится здесь форматирование разметки. Сделай, пожалуйста, красивую лесенку

`<a href="https://htmlacademy.ru/"> 
      <img src="images/logo-htmlacademy.svg" width="115" height="34" alt="Логотип HTML Academy.">
</a>`

Pakistanka reviewed (changes requested):
1. Это изображение никогда работать не будет, потому что в наименовании есть пробел - http://joxi.ru/D2Pn70NtwyZy7r, http://joxi.ru/V2V7Mg0uBRX4gA
2. Что это? Картинки должны иметь конкретные названия. Оставлять названия от дизайнера - плохая практика - http://joxi.ru/brRxNpqHBDRpM2
3. Наименование картинок с маленькой буквы  должно быть - http://joxi.ru/Dr8YwOzTK9L19m

d7add9 committed:
исправляет картинки и пр.

Пробелы между словами в названиях файла заменены на "-".
Убраны все заглавные буквы.
Названия скорректированы так, чтобы лучше отражать смысл картинки.
Исправлено форматирование: убран лишний пробел, добавлена лесенка у <a class="navigation-logo">
